public/components/custom_panels/helpers/utils.tsx

```diff
@@ -344,12 +226,11 @@ export const renderCatalogVisualization = async ({
   setVisualizationMetaData,
   setIsLoading,
   setIsError,
-  spanResolution,
-  queryMetaData,
 }: {
   http: CoreStart['http'];
   pplService: PPLService;
-  catalogSource: string;
+  savedVisualizationId: string;
+  inputMetaData: SavedVisualization;
```

Comment:
In a certain case (metric grid) the VisualizationConainer, will be passed the meta-data directly (`inputMetaData`), instead of having a saved record to render.  It is passed here for processing.